src/js/views/clinicians/sidebar/clinician-sidebar.scss

```diff
@@ -15,3 +15,14 @@
   right: 16px;
   top: 8px;
 }
+
+.clinician-sidebar__worklist {
```

Comment:
the fact that either of these domain styles need to exist make me feel like our generic sidebar styles need refactoring.  it's surprising that we would need to add styles for this generally.

Reply:
`.sidebar__section` is almost a perfect match. But it has too much top margin than is shown in Mark's designs.

For the informational text we have `.sidebar__info`. But it has a yellow background we don't want here.